libvast/src/format/pcap.cpp

```diff
@@ -106,6 +108,27 @@ const char* reader::name() const {
   return "pcap-reader";
 }
 
+vast::system::report reader::status() const {
+  using namespace std::string_literals;
+  auto stats = pcap_stat{};
```

Comment:
I think we should also make sure that we are not reading from a trace here, i.e.,  make sure we are in open live mode. `pcap_stats` probably returns an error if this is not the case, but maybe we want to log those errors as warnings and disentangle them from things that are actual errors.

Reply:
`pcap_stats` returns a non-zero exit code when not reading from an interface according to the documentation.

Reply:
Okay, so the question is: if we read from an interface and `pcap_stats` returns an error, do we want to log it (as warning) or it ignore it silently?

Reply:
I think we want to ignore it silently, as this function is very much expected fail, e.g., when not reading from an interface.